packages/medusa-file-spaces/src/services/digital-ocean.js

```diff
@@ -70,7 +70,7 @@ class DigitalOceanService extends AbstractFileService {
 
     const pass = new stream.PassThrough()
 
-    const fileKey = `${fileData.name}-${Date.now()}.${fileData.ext}`
```

Comment:
issue: By removing that we do not support uploading multiple files with the same name anymore

Reply:
which was a fix we've done ages ago

Reply:
I think we should move the responsibility of assigning timestamp into each of the batch job strategies in order to still support it, but not force the format upon other users of the fileservices